### PR TITLE
added different lucene's Similarity class implementations for experiments

### PR DESCRIPTION
**Pull request by ymatsuda**
Lucene's Similarity controls many scoring parameters. I defined a several Similarity implementaions to test TF and Coord. Coord is a weighting based on the number of matching boolean clauses.

### CODE REVIEW

File: shoebox/app/com/keepit/search/Similarity.scala [hunk 1]
```diff
@@ -0,0 +1,47 @@
+package com.keepit.search
+
+import com.keepit.common.logging.Logging
+import org.apache.lucene.search.{Similarity => LuceneSimilarity}
+import org.apache.lucene.search.DefaultSimilarity
+
+object Similarity extends Logging {
+  trait noTF { this: LuceneSimilarity =>
```
Comment (eishay) [inline shoebox/app/com/keepit/search/Similarity.scala]: why not capitalizing the trait name as we usually do?

Comment (ymatsuda) [inline shoebox/app/com/keepit/search/Similarity.scala]: Oh. I will fix it.

**Comment by andrewconner** [comment]
`tf` is term frequency, correct?

**Comment by ymatsuda** [comment]
tf is term frequency. Similary has a method, tf(freq). It converts a raw frequency to some floating point number. The default implementation of sqrt(freq).

**Comment by andrewconner** [comment]
Ah cool, thanks.

**Pull request merged**